redhat-developer/developers.redhat.com
# [DEVELOPER 5397] Create Profiling Purpose Attributes

### JIRA Issue Link
https://issues.jboss.org/browse/DEVELOPER-5397

### Verification Process
1. Create a node from a non-depreciated content type
2. See options to enter "Purpose Attributes"
3. Able to add Lists to new Vocabularies in Drupal Taxonomies which will update the available option in the Purpose Attributes field.

### Admin Screen
#### Closed
![screen shot 2018-10-04 at 11 06 01 am](https://user-images.githubusercontent.com/41014351/46490580-9fe6fc00-c7c5-11e8-9b6e-52558e752cac.png)

#### Open
![screen shot 2018-10-04 at 11 06 09 am](https://user-images.githubusercontent.com/41014351/46490578-9fe6fc00-c7c5-11e8-83db-8780133a1949.png)
